Src/VimCore/CoreInterfaces.fs

```diff
@@ -4381,6 +4381,9 @@ and IVimBufferData =
     /// The current file path for this buffer, relative to CurrentDirectory
     abstract CurrentRelativeFilePath : string option
     
+    /// The current directory for this particular window

```

Comment:
Shouldn't this comment, and the one on CurrentDirectory, reference "buffer" not "window"?

Reply:
I am constantly torn whether to use Visual Studio terminology or vim terminology. In this case, the other property used "window" so I followed that pattern.

The vim documentation for `:lcd` is:

> Like |:cd|, but only set the current directory for the current window.  The current directory for other windows is not changed.

But, yes, VsVim does not use the "window" terminology and so it should be "buffer" for consistency. Visual Studio uses buffer vs. view in this situation but VsVim calls the view a vim buffer and the buffer a text buffer. But if we say "buffer" we don't mean other vim buffers with the same underlying text buffer.